samples/vsphere/appliances/health_messages.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
```

Comment:
Looks like the logic of the sample is not complex. Consider to create a basic sample using this format:
https://github.com/vmware/vsphere-automation-sdk-python/blob/master/sample_template/sample_template_basic.py